.vuepress/config.yml

```diff
@@ -60,7 +60,7 @@ themeConfig:
     - text: Guide
       link: /guide/
     - text: Tutorials
-      link: /tutorials/
+      link: https://blog.mesg.com/tag/getting-started/
```

Comment:
there is a `tutorials` category in the menu of the blog. I think we should use it:
```suggestion
      link: https://blog.mesg.com/tag/tutorials/
```

Reply:
The tutorial article seems to not use the `tutorials` tags. It doesn't make sense to tag all of them with `getting-started`

Reply:
@EmmanuelDrouin just did it :)